v1 (#36)
cosmos/new-cosmos.network: content/articles/what-is-a-validator.md

---
index: 18
title: What is a validator?

> description: Validators are special actors in the network responsible for adding new blocks of transactions to the blockchain.

Reviewer: ```suggestion
description: Validators are special actors in the blockchain network that are responsible for adding new blocks of transactions to the blockchain.
```

  - blockchain
  - atom
---

> Validators are special actors in the network responsible for adding new blocks of transactions to the blockchain. Anyone can declare themselves as validator candidate, but only 125 candidates can enter the set of active validators. Validator candidates are chosen based on the amount of voting power associated with their operator account. Voting power is obtained when ATOM holders start staking, in proportion of the amount of ATOM staked. This voting power must then be [delegated](/learn/faq/what-is-delegating) to one or multiple validator candidates. As a result, ATOM holders will have to choose validator(s) when they stake. 

Reviewer: ```suggestion
Validators are special actors in the blockchain network that are responsible for adding new blocks of transactions to the blockchain. Anyone can declare themselves as a validator candidate, but only 125 candidates can enter the set of active validators. Validator candidates are chosen based on the amount of voting power associated with their operator account. Voting power is obtained when ATOM holders start staking in proportion to the amount of ATOM staked. This voting power must then be [delegated](/learn/faq/what-is-delegating) to one or multiple validator candidates. As a result, ATOM holders must choose one or more validators when they stake. 
```

Reviewer: let's not use (s) for plural https://developers.google.com/style/plurals-parentheses

> In order to add blocks to the blockchain, validators electronically sign block proposals that are valid according to the protocol (the proposer of a given block is selected among validators according to a specific algorithm, and rotates every block). Each signature is weighted by the voting power of the validator, and a block is considered valid if it is signed twice by more than two thirds of validators, weighted by their respective voting power.

Reviewer: ```suggestion
To add blocks to the blockchain, validators electronically sign block proposals that are valid according to the protocol. The protocol ensures that the proposer of a given block is selected among validators according to a specific algorithm and rotates every block. The voting power of the validator weights each signature. A block is considered valid if it is signed twice by more than two-thirds of validators, weighted by their respective voting power.
```
